code/datums/supplypacks.dm

```diff
@@ -1790,7 +1790,8 @@ GLOBAL_LIST_INIT(all_supply_groups, list(SUPPLY_EMERGENCY,SUPPLY_SECURITY,SUPPLY
 	cost = 10
 	special = TRUE
 	containername = "empty station goal crate"
-	containertype = /obj/structure/closet/crate/engineering
+	containertype = /obj/structure/closet/crate/secure/engineering
+	access = ACCESS_CONSTRUCTION
```

Comment:
Curios where theres a diff on `1793`. Did you change anything?

Reply:
Changed to a secure crate: `/crate/engineering` > `/crate/secure/engineering`

Reply:
My eyes actually need checking, and I guess this is what happens when you use the split git view instead of the layered one (Its much more obvious like this)
![image](https://user-images.githubusercontent.com/25063394/97723626-e8b43080-1ac3-11eb-8551-eb893199b87c.png)

Compared to this
![image](https://user-images.githubusercontent.com/25063394/97723646-ef42a800-1ac3-11eb-944f-f41e8839c27d.png)

Reply:
Resolving since this isn't an issue.